packages/react-dataparcels/src/useParcelBuffer.js

```diff
@@ -80,9 +80,11 @@ export default (params: Params): Return => {
             child: innerParcel.child
         };
 
-        return parcel._changeAndReturn(
+        let [changedParcel] = parcel._changeAndReturn(
```

Comment:
example of destructuring from something that can return null, not sure why flow doesn't complain about this?